### ISSUE TITLE
Vertical slides 'fade' instead of 'slide' while transition style is on 'slide'

### ISSUE THREAD

**Issue by GamuWasTaken** [MSzturc/obsidian-advanced-slides#314]
**Describe the bug**
Pretty much the title,
- Transition style config = 'slide'
- Expected: Vertical slide transition to be 'slide'
- Observed: Vertical slide transition is 'fade'

**Samples to Reproduce**
I don't know if its enough but, with default settings (I've just discovered them) this md:

```md
# There is a bug
---
# With vertical slides
--
# Should move but it
--
# Fades...
```

should show the bug, at least it... does on my machine 😅

**Expected behavior**

Vertical slide transition should be 'slide'

I've tested a bit with other transitions and seems to be just broken on 'slide'

**Observed behavior**
![Obsidian_eYsVva2yyD](https://github.com/user-attachments/assets/a792dc3d-0e41-4bdf-8bed-c00a277b2344)

edit:
![image](https://github.com/user-attachments/assets/3ed0c8f0-edcc-40bc-916a-06d04f551f3f)

**Comment by Diego-Paris** [comment]
Ran into this. Did you ever find a workaround?

**Comment by GamuWasTaken** [comment]
Nope, I remember searching through the extension's code, but it seems I did not find a solution
